README.md

```diff
@@ -6,11 +6,23 @@ This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next
 
 ## Getting Started
 
-First, run the development server:
+First make sure that you have corepack installed and enabled:
+
+```bash
+corepack enable
+```
+
+you can find more information here: https://github.com/nodejs/corepack
+
+After that install the dependencies with yarn
+
+```bash
+yarn install
+```
+
+Finally run the dev server
 
 ```bash
-npm run dev
```

Comment:
why did you remove npm reference?

Reply:
Unfortunately using npm will not respect the lockfile generated by yarn. The project already depends on yarn anyway so including npm there was an error from the start. Either we use yarn or we just switch the project to vanilla npm. This is pretty much by design when using yarn.